### ISSUE TITLE
cannot find value `SAPP_MODIFIER_SHIFT` in crate `sapp`

### ISSUE THREAD

**Issue by canadaduane** [not-fl3/miniquad#11]
I believe this was introduced in `0.2.7`. When I build, I now see the following errors:

```
error[E0425]: cannot find value `SAPP_MODIFIER_SHIFT` in crate `sapp`
   --> /Users/duane/.cargo/registry/src/github.com-1ecc6299db9ec823/miniquad-0.2.7/src/event.rs:277:26
    |
277 |         if value & sapp::SAPP_MODIFIER_SHIFT != 0 {
    |                          ^^^^^^^^^^^^^^^^^^^ not found in `sapp`

error[E0425]: cannot find value `SAPP_MODIFIER_CTRL` in crate `sapp`
   --> /Users/duane/.cargo/registry/src/github.com-1ecc6299db9ec823/miniquad-0.2.7/src/event.rs:280:26
    |
280 |         if value & sapp::SAPP_MODIFIER_CTRL != 0 {
    |                          ^^^^^^^^^^^^^^^^^^ not found in `sapp`

error[E0425]: cannot find value `SAPP_MODIFIER_ALT` in crate `sapp`
   --> /Users/duane/.cargo/registry/src/github.com-1ecc6299db9ec823/miniquad-0.2.7/src/event.rs:283:26
    |
283 |         if value & sapp::SAPP_MODIFIER_ALT != 0 {
    |                          ^^^^^^^^^^^^^^^^^ not found in `sapp`

error[E0425]: cannot find value `SAPP_MODIFIER_SUPER` in crate `sapp`
   --> /Users/duane/.cargo/registry/src/github.com-1ecc6299db9ec823/miniquad-0.2.7/src/event.rs:286:26
    |
286 |         if value & sapp::SAPP_MODIFIER_SUPER != 0 {
    |                          ^^^^^^^^^^^^^^^^^^^ not found in `sapp`

error: aborting due to 4 previous errors
```

(This is with target of `wasm32-unknown-unknown`)

**Comment by not-fl3** [comment]
Oh, we really need to setup CI for all the targets! 

Fixed in `0.2.8`!

**Comment by not-fl3** [comment]
oh, actually in `0.2.9`  :/ 
Updated `gwg`'s cargo.toml as well
